apps/cms/package.json

```diff
@@ -44,7 +44,6 @@
     "@graphql-codegen/typed-document-node": "3.0.2",
     "@graphql-codegen/typescript": "3.0.2",
     "@graphql-codegen/typescript-operations": "3.0.2",
-    "@graphql-codegen/typescript-urql": "3.7.3",
```

Comment:
looks like some apps are using genreated hooks. Do we have any reason to remove it? It will be tree shaked, so I think we can leave it, unless we have any other reason to remove it?

Reply:
- one package less to update or version to mismatch
- faster code generation, few apps used it
- I was exploring _what_ do we use, before switching to `client` codegen preset. 

Marked PR as draft, since it's WIP :) Did not see usages in half of our apps so pushed changes, since it was easy to do